### PR TITLE
Add Win32-network hash

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

**Comment by nfrisby** [comment]
FYI, I changed the reviewers to just @coot.

File: cabal.project [hunk 1]
```diff
@@ -139,7 +139,8 @@ package small-steps-test
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/Win32-network
-  tag: d81187239fe5f303aa1770565ad307efa605c820
+  tag: f71be820aaa480256bba24fc3f2a1e12cd1eab0f
+  --sha256: 1ggm3239z41z0ca9al43v9y2wmf20bbmdgj1k27kyph54gqkdbhn
```
Comment (coot) [inline cabal.project:143]: Use: `94153b676617f8f33abe8d8182c37377d2784bd1` tag, `f71be` is not on `master` anymore.

**Review by coot — CHANGES_REQUESTED** [review]

**Comment by newhoggy** [comment]
Updated to:

```
source-repository-package
  type: git
  location: https://github.com/input-output-hk/Win32-network
  tag: 94153b676617f8f33abe8d8182c37377d2784bd1
  --sha256: 0pb7bg0936fldaa5r08nqbxvi2g8pcy4w3c7kdcg7pdgmimr30ss
```

**Review by coot — COMMENTED** [review]
LGTM

**Review by coot — APPROVED** [review]
LGTM

**Comment by coot** [comment]
bors merge

**Pull request merged**